File: Password-Keeper-API/src/main/java/pl/kamilwojcik/passwordkeeper/exceptions/ErrorCode.java
package pl.kamilwojcik.passwordkeeper.exceptions;

public enum ErrorCode {

    /**
     * request
     */
    BAD_REQUEST,

    /**
     * auth
     */
    AUTHENTICATION_FAILED,
    BAD_CREDENTIALS,
    UNKNOWN_DEVICE,
    ACCESS_DENIED,

    /**
     * resources
     */
    NO_RESOURCE;


    public ErrorBody toErrorBody() {
        return new ErrorBody(this);
    }

}

File: Password-Keeper-WebApp2/src/logic/ErrorCode.ts
export enum ErrorCode {

    /**
     * request
     */
    BAD_REQUEST = "BAD_REQUEST",

    /**
     * auth
     */
    AUTHENTICATION_FAILED = "AUTHENTICATION_FAILED",
    BAD_CREDENTIALS = "BAD_CREDENTIALS",
    UNKNOWN_DEVICE = "UNKNOWN_DEVICE",
    ACCESS_DENIED = "ACCESS_DENIED",

    /**
     * resources
     */
    NO_RESOURCE = "NO_RESOURCE",


    UNDEFINED = "UNDEFINED"
}